src/Stateless/EntryActionBehaviour.cs

```diff
@@ -10,14 +10,15 @@ public partial class StateMachine<TState, TTrigger>
     {
         internal abstract class EntryActionBehavior
         {
-            readonly string _actionDescription;
+            MethodDescription _description;
 
-            protected EntryActionBehavior(string actionDescription)
+            protected EntryActionBehavior(MethodDescription description)
             {
-                _actionDescription = Enforce.ArgumentNotNull(actionDescription, nameof(actionDescription));
+                _description = description;
             }
 
-            internal string ActionDescription { get { return _actionDescription; } }
+            // Rename base class 'Description' for backward compatibility
```

Comment:
Is this comment accurate? The current class doesn't have a base class. Since it's not a public type, do we need to worry about compatibility here at all?